must-gather/Dockerfile

```diff
@@ -1,39 +1,39 @@
-FROM --platform=$BUILDPLATFORM quay.io/konveyor/builder:ubi9-latest AS konveyor-builder
+FROM --platform=$BUILDPLATFORM quay.io/konveyor/builder:ubi9-latest AS builder
 ARG BUILDPLATFORM
 ARG TARGETOS
 ARG TARGETARCH
-ARG RESTIC_BRANCH=konveyor-0.15.0
-ARG VELERO_BRANCH=konveyor-dev
-WORKDIR /build
-RUN curl --location --output velero.tgz https://github.com/openshift/velero/archive/refs/heads/${VELERO_BRANCH}.tar.gz && \
-    tar -xzvf velero.tgz && cd velero-${VELERO_BRANCH} && \
-    VELERO_COMMIT=$(git ls-remote https://github.com/openshift/velero HEAD | awk '{printf $1}') && \
-    CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -mod=mod -ldflags '-extldflags "-static" -X github.com/vmware-tanzu/velero/pkg/buildinfo.Version='"${VELERO_BRANCH}"' -X github.com/vmware-tanzu/velero/pkg/buildinfo.GitSHA='"${VELERO_COMMIT}" -o /velero github.com/vmware-tanzu/velero/cmd/velero && \
-    cd .. && rm -rf velero.tgz velero-${VELERO_BRANCH} && \
-    curl --location --output restic.tgz https://github.com/openshift/restic/archive/refs/heads/${RESTIC_BRANCH}.tar.gz && \
-    tar -xzvf restic.tgz && cd restic-${RESTIC_BRANCH} && \
-    CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -mod=mod -ldflags '-extldflags "-static"' -o /restic github.com/restic/restic/cmd/restic && \
-    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH}
+ARG KOPIA_BRANCH=konveyor-dev
+ARG RESTIC_BRANCH=konveyor-dev
```

Comment:
do we need to change these?

Reply:
:thought_balloon: : _FWIW, downstream builds are using oadp-1.5 release branches as source regardless of the value here._

Reply:
I can update in separate PR then to not block this one